frontend/src/components/Login.js

```diff
@@ -51,25 +50,42 @@ class Register extends Component {
       this.setState({ isLoggedIn: this.isLoggedIn() });
     } else {
       // TODO: Display message if login wasn't successful
+      this.handleAPIErrors(message);
     }
   };
 
+  handleAPIErrors(message) {
+    console.log(message);
+    this.setState({
+      errors: [
+        ...this.state.errors,
+        { message: message, key: this.state.errors.length }
+      ]
+    });
+  }
+
+  canSubmit() {
+    var canSubmit = false;
+    if (validateEmail(this.state.email)) {
+      // TODO: Add more validation
+      canSubmit = true;
+    }
+    return canSubmit;
+  }
+
   isLoggedIn() {
     const cookies = new Cookies();
     return cookies.get('jwt') !== undefined;
   }
 
   getLoggedInMessage() {
-    if (this.state.isLoggedIn) {
-      return <FormAlert>You are logged in!</FormAlert>;
-    } else {
-      return <FormAlert isRed={true}>You are not logged in!</FormAlert>;
-    }
+    return this.state.errors.map(({ message, key }) => {
+      return <FormAlert key={key}>{message}</FormAlert>;
+    });
   }
 
   render() {
     const message = this.getLoggedInMessage();
```

Comment:
Style thing -- in the case of functions that return components, my practice has been to put the function inside `render`. That is:

```javascript
render() {
  return (
    <div>
      {this.renderSomePart()}
    </div>
  );
}
```